scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -702,14 +702,30 @@
                         {
                             "inPoint": 12000,
                             "outPoint": 15550,
-                            "text": "There once was a little wooden boy.",
-                            "label": "JHC"
+                            "text": {
+                                "items": [
+                                    {
+                                        "languageId": "eng",
+                                        "role": "original",
+                                        "text": "There once was a little wooden boy."
+                                    }
+                                ]
+                            },
+                            "speakerInitials": "JHC"
                         },
                         {
                             "inPoint": 18300,
                             "outPoint": 19240,
-                            "text": "His name was Pinocchio",
-                            "label": "JHC"
+                            "text": {
+                                "items": [
+                                    {
+                                        "languageId": "eng",
```

Comment:
We must make the timestamp consistency rules invariant validation rules to validate pre-seeded snapshot test data properly.

Reply:
Let's add this as a tech-debt story.

Reply:
[Here](https://www.pivotaltracker.com/story/show/184431225) is the story.